weaver/samples/corda/corda-simple-application/makefile

```diff
@@ -55,23 +55,31 @@ test: github.properties
 .PHONY: initialise-vault
 initialise-vault:
 	(docker logs corda_partya_1 > /dev/null && REMOTE_FLOW="localhost:30006#com.cordaSimpleApplication.flow.GetStateByKey:*" ./clients/build/install/clients/bin/clients configure all) || true
-	(docker logs corda_network2_partya_1 > /dev/null && CORDA_PORT=30006 NETWORK_NAME=Corda_Network2 REMOTE_FLOW="localhost:10006#com.cordaSimpleApplication.flow.GetStateByKey:*" ./clients/build/install/clients/bin/clients configure all) || true
+	if [ $(shell docker ps | grep corda_network2_partya_1 | wc -l) -eq 1 ]; then \
```

Comment:
Just to clarify `docker logs corda_network2_partya_1 > /dev/null && ` does check whether `corda_network2_partya_1` is running or not, and if it is running only then the subsequent command runs.

Does this `if` condition do anything new? If not then both are redundant checks, we can leave it the way it was?

Reply:
I want to avoid displaying error messages which may confuse users, even if those errors don't cause any harm.

I did a more comprehensive pass through the scripts and replaced the logging logic with conditionals.